live-build/misc/ansible-roles/appliance-build.zfsonlinux-development/tasks/main.yml

```diff
@@ -76,3 +77,14 @@
     group: staff
     state: directory
     recurse: yes
+
+- file:
+    path: "/export/home/delphix/.cargo/"
+    state: directory
+    owner: delphix
+    group: staff
+- copy:
+    dest: "/export/home/delphix/.cargo/config.toml"
```

Comment:
Is there a system wide config we could use? I'm OK with this, just curious..

Reply:
I think the only option for a global configuration is to put something in `/.cargo/config.toml`, which feels weird to me, but apparently cargo checks all directories up the hierarchy to the root. I can change to that if people prefer it but putting it in the home directory is more idiomatic, I think.

Reply:
OK, yea this way sounds good.